frontend/src/components/CreateGoal/ObjectiveForm.js

```diff
@@ -0,0 +1,139 @@
+  objective,
+  setObjective,
+  errors,
+  topicOptions,
+}) {
+  // the parent objective data from props
+  const { title, topics, resources } = objective;
+
+  // onchange handlers
+  const onChangeTitle = (e) => setObjective({ ...objective, title: e.target.value });
+  const onChangeTopics = (newTopics) => setObjective({ ...objective, topics: newTopics });
+  const setResources = (newResources) => setObjective({ ...objective, resources: newResources });
+
+  // validate different fields
+  const validateObjectiveTitle = () => {
+    if (!title) {
+      const newErrors = [...errors];
+      newErrors.splice(OBJECTIVE_FORM_FIELD_INDEXES.TITLE, 1, <span className="usa-error-message">{objectiveTitleError}</span>);
+      setObjectiveError(index, newErrors);
+    } else {
+      const newErrors = [...errors];
+      newErrors.splice(OBJECTIVE_FORM_FIELD_INDEXES.TITLE, 1, <></>);
+      setObjectiveError(index, newErrors);
+    }
+  };
+
+  const validateObjectiveTopics = () => {
+    if (!topics.length) {
+      const newErrors = [...errors];
+      newErrors.splice(OBJECTIVE_FORM_FIELD_INDEXES.TOPICS, 1, <span className="usa-error-message">{objectiveTopicsError}</span>);
+      setObjectiveError(index, newErrors);
+    } else {
+      const newErrors = [...errors];
+      newErrors.splice(OBJECTIVE_FORM_FIELD_INDEXES.TOPICS, 1, <></>);
+      setObjectiveError(index, newErrors);
+    }
+  };
+  const validateResources = () => {
+    let error = <></>;
+
+    const validated = validateListOfResources(resources);
+
+    if (!validated) {
+      error = <span className="usa-error-message">{objectiveResourcesError}</span>;
+    }
+
+    const newErrors = [...errors];
+    newErrors.splice(OBJECTIVE_FORM_FIELD_INDEXES.RESOURCES, 1, error);
+    setObjectiveError(index, newErrors);
+  };
+
+  return (
+    <div className="margin-top-2">
+      <div className="display-flex flex-justify maxw-mobile-lg">
+        <h3>Objective summary</h3>
+        <Button type="button" unstyled onClick={() => removeObjective(index)} aria-label={`Remove objective ${index + 1}`}>Remove this objective</Button>
+      </div>
+      <FormGroup className="margin-top-1" error={errors[OBJECTIVE_FORM_FIELD_INDEXES.TITLE].props.children}>
+        <Label htmlFor="objectiveTitle">
+          TTA objective
+          {' '}
+          <span className="smart-hub--form-required font-family-sans font-ui-xs">*</span>
+        </Label>
+        {errors[OBJECTIVE_FORM_FIELD_INDEXES.TITLE]}
+        <Textarea id="objectiveTitle" name="objectiveTitle" required value={title} onChange={onChangeTitle} onBlur={validateObjectiveTitle} />
+      </FormGroup>
+      <FormGroup error={errors[OBJECTIVE_FORM_FIELD_INDEXES.TOPICS].props.children}>
+        <Label htmlFor="topics">
+          Topics
+          {' '}
+          <span className="smart-hub--form-required font-family-sans font-ui-xs">*</span>
+        </Label>
+        {errors[OBJECTIVE_FORM_FIELD_INDEXES.TOPICS]}
+        <Select
```

Comment:
In the activity report we have the topics multi-select stay open after a selection. Do we want to match that functionality here?